src/compounds/phone-number-modal/src/index.tsx

```diff
@@ -1,154 +1,177 @@
 /** @jsx jsx */
-
-import * as React from 'react'
+import React from 'react'
 import { jsx } from 'theme-ui'
-import { action } from '@storybook/addon-actions'
 import { ImgixImage } from '@uswitch/trustyle.imgix-image'
 import Modal from '@uswitch/trustyle.modal'
 
-interface Props extends React.HTMLAttributes<HTMLDivElement> {
-  logo: string
-  logoDescription: string
-  imageCritical?: boolean
+type setStateClosed = () => void
+
+interface PhoneNumberModalInfo {
   phoneNumber: string
-  openingTimes?: any
-  termsAndConditions?: string
+  logoUrl: string
+  logoDescription: string
+  termsAndConditions: string
+  openingTimes?: string[]
   url?: string
-  complianceText: any
+  complianceText: string[]
+}
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  phoneNumberModalInfo: PhoneNumberModalInfo
+  logoImageCritical?: boolean
+  isOpen: boolean
+  modalAriaLabel: string
+  setStateClosed: setStateClosed
```

Comment:
```javascript
setStateClosed: () => void
```

or update the type to be capitalized (`SetStateClosed`)

Reply:
Changed this, thanks.